great_expectations/rule_based_profiler/parameter_builder/partition_parameter_builder.py

```diff
@@ -66,22 +66,20 @@ def __init__(
             data_context: BaseDataContext associated with this ParameterBuilder
         """
 
-        self._column_partition_metric_single_batch_parameter_builder_config = (
-            ParameterBuilderConfig(
-                module_name="great_expectations.rule_based_profiler.parameter_builder",
-                class_name="MetricSingleBatchParameterBuilder",
-                name="column_partition_metric_single_batch_parameter_builder",
-                metric_name="column.partition",
-                metric_domain_kwargs=DOMAIN_KWARGS_PARAMETER_FULLY_QUALIFIED_NAME,
-                metric_value_kwargs={
-                    "bins": "auto",
-                    "allow_relative_error": False,
-                },
-                enforce_numeric_metric=False,
-                replace_nan_with_zero=False,
-                reduce_scalar_metric=False,
-                evaluation_parameter_builder_configs=None,
-            )
+        self._column_partition_metric_single_batch_parameter_builder_config = ParameterBuilderConfig(
+            module_name="great_expectations.rule_based_profiler.parameter_builder",
+            class_name="MetricSingleBatchParameterBuilder",
+            name="column_partition_metric_single_batch_parameter_builder",
+            metric_name="column.partition",
+            metric_domain_kwargs=DOMAIN_KWARGS_PARAMETER_FULLY_QUALIFIED_NAME,
+            # metric_value_kwargs={
```

Comment:
Should we just delete this?

Reply:
@billdirks If we keep this, we can easily satisfy the general needs of this parameter builder, while making it customizable for the `OnboardingDataAssistant` (and other applications).  Thanks.